### DISCUSSION TITLE
Using with pi-hole

### DISCUSSION THREAD

**Discussion by shadyxN8ANacSoBCJzZpjG7LyXNgNfMAbLoME** [wg-easy/wg-easy#1393] [Q&A]
I followed this:
https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Pi-Hole

I am using pi-hole and wg-easy. I am having traffic address not matching in pi-hole.
My phone is setup to use Wireguard and has been given the address: `10.8.1.2/24`
In pi-hole, I can easily see my phone's requests coming in at the address: `10.8.1.1`
Now if I add another device to Wireguard, it also has the same address inside pi-hole.

I would like each client to have the same address listed in the wireguard config to show up in pi-hole.

Below is my docker-compose.yml
Note that for `WG_DEFAULT_DNS` I had to change the address from the docker pi-hole address `10.8.1.100/24` to the actual local ip address of the physical machine: `192.168.50.69` for any internet to work. There are no other instances of pi-hole running

```
services:  
 wg-easy:  
   restart: unless-stopped  
   environment:  
     - WG_HOST=public.ip.address.
     - PASSWORD_HASH=hashed_password 
     - WG_DEFAULT_DNS=10.8.0.100  
     - WG_DEFAULT_ADDRESS=10.8.0.x  
     - LANG=en  
     - WG_PORT=51820
     - PORT=51821
     - UI_TRAFFIC_STATS=true
   image: ghcr.io/wg-easy/wg-easy
   container_name: wgeasy_stack
   volumes:
     - /home/user/pihole_wgeasy/wgeasy:/etc/wireguard
   ports:
     - "51820:51820/udp"
     - "192.168.50.69:51821:51821/tcp"
   cap_add:
     - NET_ADMIN
     - SYS_MODULE
   sysctls:
     - net.ipv4.ip_forward=1
     - net.ipv4.conf.all.src_valid_mark=1
   networks:
     wgeasy_pihole:
       ipv4_address: 10.8.1.200
  
 pihole:
   image: pihole/pihole
   container_name: pihole_stack
   restart: unless-stopped
   environment:
     - WEBPASSWORD=password
     - TZ=TZ
     - IPv6=false
     - DNS1=9.9.9.9
     - DNS2=149.112.112.112
   volumes:
     - /home/user/pihole_wgeasy/pihole/etc-pihole:/etc/pihole
     - /home/user/pihole_wgeasy/pihole/etc-dnsmasq.d:/etc/dnsmasq.d
     - /home/user/pihole_wgeasy/pihole/resolv.conf:/etc/resolv.conf
     - /home/user/pihole_wgeasy/pihole/pihole-FTL.conf:/etc/pihole/pihole-FTL.conf
   ports:  
     - "192.168.50.69:53:53/tcp"  
     - "192.168.50.69:53:53/udp"  
     - "8800:80/tcp"  
   networks:  
     wgeasy_pihole:  
       ipv4_address: 10.8.1.100  
  
networks:  
 wgeasy_pihole:
   ipam:  
     config:  
       - subnet: 10.8.1.0/24
```

**Comment by kaaax0815** [comment]
`WG_DEFAULT_DNS=10.8.0.100` would have to be `WG_DEFAULT_DNS=10.8.1.100`
Duplicate of https://github.com/wg-easy/wg-easy/discussions/1028 and https://github.com/wg-easy/wg-easy/discussions/94
